cmd2_abbrev/abbrev.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, *args, **kwargs):
         # code placed here runs after cmd2 initializes
         # this is where you register any hook functions
         self.abbrev = False
-        self.settable.update({'abbrev': 'Accept command abbreviations'})
+        self.settables.update({'abbrev': 'Accept command abbreviations'})
```

Comment:
There is a new `add_settable` method and `Settable` class which should be used like so:
```Python
self.add_settable(cmd2.Settable('abbrev', bool, 'Accept command abbreviations'))
```

Reply:
sure, fixed it.

It is a really useful plugin though :)